internal/querycoordv2/checkers/channel_checker.go

```diff
@@ -91,7 +91,7 @@ func (c *ChannelChecker) Check(ctx context.Context) []task.Task {
 		}
 	}
 
-	channels := c.dist.ChannelDistManager.GetAll()
+	channels := c.dist.ChannelDistManager.GetByFilter(nil)
```

Comment:
pass not filter is good enough
```suggestion
	channels := c.dist.ChannelDistManager.GetByFilter()
```